doc/users/installing/index.rst

```diff
@@ -281,11 +281,11 @@ be manually installed on Macports with ::
 Once pip is installed, you can install Matplotlib and all its dependencies with
 from the Terminal.app command line::
 
-   python3 -mpip install matplotlib
+   python3 -m pip install matplotlib
 
 (``sudo python3.6 ...`` on Macports).
```

Comment:
Some Mac person (@greglucas I have identified as one) can probably tell if there is a better thing than `3.6` to write here.

Reply:
I now realize that the whole section is about OSX, so probably I should stop reviewing... 

But my, not very qualified, guess is that you are doing the right thing.

Reply:
After reading up a bit:

* Both `-mpip` and `-m pip` works, but consistency is nice.
* OSX comes with 2.7 as default `python` so for the OSX parts it should be `python3`.

Reply:
I didn't realize -mpip was valid. 

In that case, maintainers can decide whether consistency warrants a change. I'm OK with it either way.

@oscargus Thank you for looking into both of those issues!

Reply:
Let's always use `-m pip`. While I personally always leave out the space when typing, separating the parameter and it's argument is more clear.